templates/app/client/components/auth(auth)/router.decorator.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-import _ from 'lodash';
+import * as _ from 'lodash';
```

Comment:
Hmm, I believe that's where Babel & TypeScript differ with modules. I believe this is the same reason why I had to use `require`s for angular modules when using TS. It works with Babel.